language-plutus-core/plutus-ir-test/types/ifError

```diff
@@ -0,0 +1,13 @@
+[ (lam b (all a (type) (fun (fun (con integer) a) (fun (fun (con integer) a) a)))
```

Comment:
As discussed, it can be a problem with golden file generation. We need extra machinery to maintain multiple golden files inside one directory. One solution I suggested is to use symlinked test files, but this may have other problems (git?).
I propose to postpone "deduplication" in another PR.